help/de/Help_ch_GUIMenus.xml

```diff
@@ -1131,7 +1149,7 @@
                 </entry>
 
                 <entry namest="c3">
-                  <para>Erstellt eine neue internationale europäische (SEPA) Transaktion per Online Banking.
+                  <para>Erstellt eine internationale europäische (SEPA) Überweisung per Onlinebanking.
```

Comment:
Für später: Wie international sind SEPA-Überweisungen?

- Verfügbarkeit: Nur in SEPA-Länder, alle Länder mit IBAN, was dazwischen?
- Kosten: Unterschied SEPA-Raum (nicht teurer als Inland) und Rest?
- analog: Laufzeit

möglische Quellen: Bank-berater, -Website, …

Reply:
@fellen: ich halte es nicht für Richtig, in einer technischen Hilfebeschreibung auf Vor- und Nachteile, Kosten usw. von Finanzprodukten einzugehen.
Ich werde den Text hier auf den Tooltip reduzieren.

Reply:
Die Frage entstand, weil ich `internationale europäische (SEPA)` für unklar formuliert halte. Als Anfänger könnte ich  denken:

- schade, Inlandsüberweisungen gehen nicht.
- Oder wg. `International` könnte ich denken, in die USA geht auch, aber die haben keine IBAN
- Brasilien hat eine SWIFT-registrierte IBAN-Struktur, geht aber wohl noch nicht …